Sources/eppo/Obfuscation.swift

```diff
@@ -21,6 +21,13 @@ func getMD5Hex(_ value: String) -> String {
     return digestData.map { String(format: "%02hhx", $0) }.joined()
 }
 
+func base64Encode(_ value: String) -> String {
```

Comment:
is this change necessary to include in this diff?

Reply:
It was to support additional testing but is not strictly needed.